README.md

```diff
@@ -1,4 +1,6 @@
-"# spec-TemplateForSimplifierProjects"
+# Installation
```

Comment:
Lass uns damit anfangen, wofür das Repo eigentlich da ist:
# About 
This template standardizes the development of fhir resources with .fsh resources and provides the following features.
* best practices for modeling fsh files (see /Resources/input/fsh folder)[link to folder].
* scripted creation and validation of fsh files and the generated fhir resources using a Hapi validator.
* sorting of validation results by severity in subfolders
* Validation of committed fsh files via github-action with different validators (hapi, simplifier).

Reply:
Sollen wir noch Ansprechpartner hinzufügen oder ergeben die sich aus den contributors für das template repo?

Reply:
scripted creation and validation of fsh files and the generated fhir resources using a Hapi validator. (See Usage[Link to usage])